Resources/Prototypes/DeltaV/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -63,4 +63,23 @@
     gloves: ClothingHandsGlovesCombat
     shoes: ClothingShoesSlippers
     id: SyndiPDA
-  innerClothingSkirt: ClothingUniformJumpsuitPyjamaSyndicatePink
\ No newline at end of file
+  innerClothingSkirt: ClothingUniformJumpsuitPyjamaSyndicatePink
+  
+# Syndicate Paramedic
+
+- type: startingGear
+  id: SyndicateParamedicGear
+  equipment:
+    jumpsuit: UniformScrubsColorCybersun
+    back: ClothingBackpackDuffelSyndicateFilledMedical
+    shoes: ClothingShoesChameleonNoSlips
```

Comment:
Is there a reason they start with no-slips instead of the regular shoes?

Reply:
Ah, so to clarify on this one. When I had originally requested that this PR be made, NoSlips were intended to be part of the outfit for, "To keep the paramedic from slipping in all the bloodstains". They aren't necessarily an essential part of the character, and really you can't slip in blood anyway, so they're here for a subtle bit of flavor. They could just as easily be replaced with ordinary black shoes that will also match the outfit.

Reply:
It's essentially just yet another free item from the uplink

Reply:
I'll make this an official request then. Let's swap the no-slips out for regular black shoes. They aren't essential to have anyway.